### PR TITLE
For #9384, updated README and polished report.

### PR DESCRIPTION
**Pull request by gplsteph**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,6 +63,17 @@ Example:
 sg-otio write -u URL --session-token TOKEN --entity-type Cut --entity-id CUT_ID --file INPUT.edl --movie INPUT.mov --settings SETTINGS.JSON
 ```
 
+### Comparing a Video Track to a SG Cut
+Read a Video Track from an OpenTimelinio source and compare it to an existing SG Cut.
+Any format suppported by OpenTimelineIO's standard adapters is supported for the source.
+The video Track can be written to SG as a new Cut by adding the `--write` argument.
```
Comment (ksallee) [inline README.md:69]:
```suggestion
The video Track can be written to SG as a new Cut by adding the `--write` argument.
If the `--write` argument is not provided, the result of the comparison is returned.
```

**Review by ksallee — COMMENTED** [review]
LGTM, although I just want to make sure that the settings and their default values provided in the README are still up to date? It seems to me you added and changed things for omit statuses.

File: README.md (hunk 1 above)
Comment (gplsteph) [reply README.md:69]: Hmm not really. The result of the comparison is written to the output, regardless of the `--write` argument.

**Comment by gplsteph** [comment]
good point, please have another look.

**Pull request merged**